core/platform/models.py

```diff
@@ -239,7 +239,7 @@ class Registry(object):
     """
 
     # Maps platform names to the corresponding module registry classes.
-    _PLATFORM_MAPPING = {
+    PLATFORM_MAPPING = {
```

Comment:
Why change this?

Reply:
H @lilithxxx Sean Lip recommended I changed the name of _Platform to Platform so it could be tested, and for consistency in the codebase, I changed this line to match the new mode name.

Reply:
But this variable is private -- i.e its not used anywhere else so I think it is better to prepend `_` to it. Thanks

Reply:
Done